cip-0047/cip-0047.md

```diff
@@ -1,7 +1,7 @@
 <pre>
   CIP:  CIP 0047
   Title: Featured App Activity Markers
-  Author: Moritz Kiefer
+  Author: Moritz Kiefer, Simon Meier
```

Comment:
Thx. Note that Wayne stated on my CIP that the authors need to be separated by a newline.